src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -504,6 +530,59 @@ private void initComboBoxes() {
         });
     }
 
+    /*
+     * filters the items of `comboBox` with `predicate` and
+     * selects the first resulting or if none, the first of the `comboBox` items
+     */
+    private <T> void selectItem(final ComboBox<T> comboBox, Predicate<T> predicate) {
+        final T item = comboBox.getItems().stream()
+                .filter(predicate)
+                .findFirst()
+                .orElse(comboBox.getItems().get(0));
+
+        comboBox.getSelectionModel().select(item);
+    }
+
+    private void selectJobBoxItemForJob(final ComboBox<PackageItem> jobBox, final String jobId) {
+        selectItem(jobBox, jobItem ->
+                jobItem.versionItems.stream()
+                        .anyMatch(vItem -> vItem.linkedPackageProperty.get().getId().equals(jobId)));
+    }
+
+    /**
+     * TODO: Reduce boilerplate code after switching to >= Java 9
```

Comment:
This will show up as JavaDoc comment - maybe not what was intended here? 

- `/** ... */` -> JavaDoc comment for method, class, ...
- `/* ... */` -> multi-line comment, not related to JavaDoc
- `// ...` ->  line comment, also not related to JavaDoc